website/docs/cloud-docs/api-docs/oauth-clients.mdx

```diff
@@ -156,7 +162,8 @@ curl \
       "http-url": "https://github.com",
       "api-url": "https://api.github.com",
       "key": null,
-      "rsa-public-key": null
+      "rsa-public-key": null,
+      "organization-scoped": false
```

Comment:
@robin-norwood I went back and checked with the team. It is actually possible for organization-scoped to be true and for there to be projects in the relationships too. 
Here is an example of when that could happen:
1. oauth-client is initially organization-scoped
2. The user adds Project A to the oauth-client and now the organization-scoped is false
3. The user sets provider back to organization-scoped, organization-scoped is now true again, **but** we want to retain all the projects that the user added to the oauth-client incase they want it to be project scoped again and so they dont have to re-add all the projects.

Reply:
Ah, got it. So, if I understand it correctly, if organization-scoped is true, the oauth client will be available to all projects in the org. If it is false, it will only be available to the projects in the `projects` list.

Reply:
yup exactly